leetcode/views/result.py

```diff
@@ -82,6 +82,10 @@ def make_failed_view(self):
                 blank, your_answer_header, your_answer,
                 blank, expected_answer_header, expected_answer
         ]
+        if len(self.result.get('std_output', '')) > 0:
+            stdout_header = urwid.Text('Stdout:')
+            stdout = urwid.Text(self.result['std_output'])
+            list_items.extend([blank, stdout_header, stdout])
```

Comment:
Is the std_output supposed to display at other result?

Reply:
Typically std_output is interesting while debugging failed tests. We can add it to other views as well but I don't think that's necessary.

Reply:
I tried to output some content to std output in my code, the "std_output" can be very large sometime that I don't think it useful to display them all. Maybe we should restrict the std_output size?

Reply:
That's a good idea. PR updated.